components/ApiUserDashboard.js

```diff
@@ -34,9 +35,15 @@ class ApiUserDashboard extends Component {
         }
         <h3>{apiUser.appName} API Keys</h3>
         <Alert variant='info'>
-          Note: copy/paste the API key values below for use in
-          the <code>x-api-key</code> header when making any request to the RMCE
-          API.
+          <Alert.Heading>
+            API Key Instructions
+          </Alert.Heading>
+          <p>
+            Below you will find your API keys to make requests to the trip planner.
+            These keys should only be used for testing because the request limit
+            is heavily restricted.
+          </p>
+          <EmailForApiKeyMessage />
```

Comment:
Not sure why this is a separate component (it puts the above text regarding limits out of context).

Reply:
I'd suggest to inline that content, or place a comment saying why it shouldn't be inlined.